src/hb-ot-layout.cc

```diff
@@ -1047,6 +1055,48 @@ hb_ot_layout_get_size_params (hb_face_t    *face,
 }
 
 
+/**
+ * hb_ot_layout_get_stylistic_set_label:
+ *
+ * Since: REPLACEME
+ **/
+int
+hb_ot_layout_get_stylistic_set_label (hb_face_t    *face,
+				      hb_tag_t      tag)
+{
+  const OT::GSUB &gsub = _get_gsub (face);
+
+  unsigned int num_features = gsub.get_feature_count ();
+  for (unsigned int i = 0; i < num_features; i++)
+  {
+    if (tag == gsub.get_feature_tag (i))
+    {
+      const OT::Feature &f = gsub.get_feature (i);
+      return f.get_feature_params ().get_stylistic_set_name_id (tag);
+    }
+  }
+
+  return -1;
+}
+
+/**
+ * hb_get_name_entry:
+ *
+ * Since: REPLACEME
+ **/
+int
+hb_get_name_entry (hb_face_t    *face,
```

Comment:
I think this should be done in a separate PR, returning name ID here should be enough for many clients, and an API for accesing name tables should be good on its own independent of this PR.